src/components/ADempiere/Form/VPOS/Collection/index.vue

```diff
@@ -495,6 +505,9 @@ export default {
     },
     fieldsPaymentType() {
       return this.fieldsList[2]
+    },
+    overUnderPayment() {
+      return this.$store.state['pointOfSales/payments/index'].dialogoInvoce.success
```

Comment:
please change state name to `dialogInvoice` (without the `o` at the end of the dialog)